bin/update-contributors.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+cat > CONTRIBUTORS.md <<EOS
+# Cumulus Contributors
+
+EOS
+
+(
+  echo 'Alireza Jazayeri'
+  git log |\
+    egrep '^Author:' |\
+    sed -e 's/^Author: //' -e 's/ <.*//' |\
+    grep ' ' |\
+    sort |\
+    uniq
+) |\
+  egrep -v '(Marourane|Philip Osip)' |\
```

Comment:
I'm assuming this is a vanity exception?    Can we comment this so it's obvious *why* that's there?